orgs/troms.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "./schema-validation.json",
+  "orgId": "troms",
+  "orgLineIdPrefix": "TRO:Line:1_",
+
+  "supportEmail": "",
+
+  "fylkeskommune": {
+    "logoSrc": "/troms_fylkeskommune.svg",
```

Comment:
IIRC this was a request from NFK as well, but Troms may want to have both the logo for fylkeskommune and the logo for Fylkestrafikk, like they have on their current page:
![2024-03-06 at 12 29 48](https://github.com/AtB-AS/planner-web/assets/14045515/3eada817-43a5-4a72-8902-0969b29060aa)

Reply:
True! Troms Fylkestrafikk is undergoing a rebranding which will result in changes to their name, logo, etc. I don't think we should put any work into this until the branding is done as this could change, but it might be something we should do later 👍

Reply:
Agree!